sentry-samples/sentry-samples-android/src/main/AndroidManifest.xml

```diff
@@ -64,7 +64,7 @@
         <!--    <meta-data android:name="io.sentry.anr.enable" android:value="false" />-->
 
         <!--    how to disable the auto-init-->
-        <meta-data android:name="io.sentry.auto-init" android:value="false" />
+        <!--    <meta-data android:name="io.sentry.auto-init" android:value="false" />-->
```

Comment:
This feature conflicts with the auto install? Curious why this change in this PR

Reply:
similar to https://github.com/getsentry/sentry-java/pull/1936#discussion_r826381968